src/lib/bootloader/config_dialog.rb

```diff
@@ -26,6 +26,8 @@ def initialize(initial_tab: :boot_code)
 
     def run
       guarded_run
+    rescue Yast::AbortException
+      :abort
```

Comment:
If you want to introduce new general Abort Exception then I find having it handled in each client a bit error prone and over kill. Why not handle it in WFM.Call itself like other general exceptions? - https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/yast/wfm.rb#L358 Just do not mix AbortException from ruby and Yast::AbortException...maybe better name like UserAbort or UIAbortException would be better to distinguish,

Reply:
I have checked it today. About allowing `WFM.Call` to handle the situation, you're completely right @jreidinger. In fact, looks like `Yast::AbortException` was introduced for that (or similar) purpose. See https://github.com/yast/yast-ruby-bindings/pull/215.

Thanks a lot!